src/modules/test/recipes/SecurityBaselineTests.json

```diff
@@ -1251,14 +1251,6 @@
     "ComponentName": "SecurityBaseline",
     "ObjectName": "remediateEnsureSmbWithSambaIsDisabled"
   },
-  {
```

Comment:
why are we no longer testing unload/load?

Reply:
This was restored. That's not to test unload/reload, is to allow remediation to be applied in full (including sshd restart). It was removed to measure for the entire run. Not necessary to remain.